test/bin/mdx-test/expect/multilines-mli/test-case.mli

```diff
@@ -0,0 +1,45 @@
+(**
+
+In OCaml docstring everything is indented with two spaces
+
+Test multi-lines shell commands:
+
+{@sh[
+  $ for i in `seq 1 10`; do \
+  >   echo $i; \
+  > done
+  1
+  ...
+  10
+]}
+
+This works for normal OCaml fragments:
+
+{[
+  let rec fact = function
+  | 1 -> 1
+  | n -> n * fact (n-1)
+]}
+
+The formatting for multilines in .mli files is the following:
+- The first line is indented two spaces after the comment opening
+- The other lines are indented to keep the original indentation relative to the
+  first line
+
+  {[
+    match None with
+    | None -> ()
+    | Some a -> match a with
+      | None -> ()
+      | Some b -> b
+  ]}
+
+But it does not work fine for toplevel (see [multilines/test-case.md] for the
```

Comment:
I don't think we should have a test making sure things fail unexpectedly. It makes more sense to have such a test as part of a test case fixing the issue.

Reply:
You are right. I sometimes read the tests to understand how a binary/library work, and can be frustrated when only the small working subset is tested (not covering my failure), but an issue will be better to track the problem.

Reply:
I definitely agree with you, this should be fixed. But I find tests that make sure the code fails to be more confusing than helpful, as they make me wonder: should this fail? Kind of like you have unit tests, you fix something and suddenly a test fails because it was expecting the value to be wrong which makes you wonder whether you fixed it or you broke it.